src/amplitude.js

```diff
@@ -290,6 +290,21 @@ define(function (require) {
     }
   };
 
+/**
+   *  bufferSize of Amplitude 
+   *  @method setBufferSize
+   *  @for p5.Amplitude
+   *  @param {Number} set bufferSize to any value multiple of 2
+   */
+
+  p5.Amplitude.prototype.setBufferSize = function(s) {
+    if (s%2==0) {
```

Comment:
it's a good practice to use triple equals so that we check for type as well

Reply:
actually i have modified it in my local repo , i will be pushing it with your recommendations keeping in mind  :smiley: